src/Temporalio/Worker/TemporalWorkerOptions.cs

```diff
@@ -102,6 +107,70 @@ public TemporalWorkerOptions(string taskQueue)
         /// </summary>
         public TimeSpan GracefulShutdownTimeout { get; set; } = TimeSpan.Zero;
 
+        /// <summary>
+        /// Gets or sets the number of workflows cached for sticky task queue use. If this is 0,
+        /// sticky task queues are disabled and no caching occurs.
+        /// </summary>
+        public int MaxCachedWorkflows { get; set; } = 10000;
+
+        /// <summary>
+        /// Gets or sets the maximum allowed number of workflow tasks that will ever be given to
+        /// the worker at one time.
+        /// </summary>
+        public int MaxConcurrentWorkflowTasks { get; set; } = 100;
+
+        /// <summary>
+        /// Gets or sets the maximum number of local activities that will ever be given to this
+        /// worker concurrently.
+        /// </summary>
+        public int MaxConcurrentLocalActivities { get; set; } = 100;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the worker will only handle workflows and local
+        /// activities.
+        /// </summary>
+        public bool DisableRemoteActivities { get; set; }
```

Comment:
Hrmm, this is what it's called in Python. I assumed "remote" means "non-local". You're saying this should be called `DisableNonLocalActivities`? Maybe I should change to Go and Java wording of `LocalActivityWorkerOnly`?

Reply:
I think `DisableNonLocalActivities` or better yet `EnableNonLocalActivities` is the clearest.
`LocalActivityWorkerOnly` isn't technically correct since the worker runs workflows (which is why Spencer renamed it in Core).

Reply:
We would prefer false to be the default IMO (like Go, proto, unset .NET boolean primitive, etc). So, what should we name it in the interest of clarity and consistency and default-false semantics? `DisableNonLocalActivities` is an inconsistent, ugly double negative.

In general, can you provide an adjective for "non-local" that Temporal can use company wide? Why can "remote" not be it since it's the opposite of local?

> LocalActivityWorkerOnly isn't technically correct since the worker runs workflows

It's only incorrect depending on how you read it. You read "local-activity worker only" and I read it as "local activity-worker only" completely unrelated to workflow workers. In fact, most of our worker options apply to either activity workers or workflow workers, not both.

Reply:
If you prefer false for default I would avoid the double negative.

I think I called non-local, remote activities a couple of years ago in the TS SDK but I haven't seen that concept anywhere in our docs.

`LocalActivityWorkerOnly` works for me or `DisableActivityPolling` (even though the latter doesn't account for eager activities), @Sushisource had issues with the local only wording.

Reply:
:+1: We use the word "remote" in Rust and Python too. If @Sushisource doesn't have any objection, I'll switch to `LocalActivityWorkerOnly` even though a tad confusing because it at least matches Go and Java.

Reply:
Fixed in next PR